tracing/tracing-test.sh

```diff
@@ -76,7 +76,7 @@ get_jaeger_status()
 
 		[ "$ret" -eq 0 ] && [ -n "$status" ] && break
 
-		attempt=$((attempt++))
+		attempt=$((attempt+1))
```

Comment:
`((attempt++))` is be enough, isn't it?

Reply:
no, it doesn't do the work

Reply:
Hmm, I use that idom elsewhere:
https://github.com/kata-containers/tests/blob/master/metrics/density/docker_memory_usage.sh#L74
```
(( count++ ))
```

Maybe it needs the spaces to invoke the commands `((` and `))` rather than maybe using a shell builting. No big matter for the merge, the intent is still clear :-)

Reply:
@chavafg - might be good to search the rest of the repo/scripts to see if we have the same issue lurking anywhere else though ;-)